src/Listeners/UpdateSplitTitleAfterDiscussionWasRenamed.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
+* This file is part of flagrow/flarum-ext-split.
+*
+* Copyright (c) Flagrow.
+*
+* http://flagrow.github.io
+*
+* For the full copyright and license information, please view the license.md
+* file that was distributed with this source code.
+*/
+
+namespace Flagrow\Split\Listeners;
+
+use Illuminate\Contracts\Events\Dispatcher;
+use Flarum\Event\DiscussionWasRenamed;
+
+use Flarum\Core\Repository\PostRepository;
+
+class UpdateSplitTitleAfterDiscussionWasRenamed {
+    /**
+     * @param Dispatcher $events
+     */
+    public function subscribe(Dispatcher $events)
+    {
+        $events->listen(DiscussionWasRenamed::class, [$this, 'whenDiscussionWasRenamed']);
+    }
+
+    /**
+     * @param \Flarum\Event\DiscussionWasRenamed $event
+     */
+    public function whenDiscussionWasRenamed(DiscussionWasRenamed $event) {
+            
+        $this->posts = app(PostRepository::class);
+        $allPosts = $this->posts->
+                query()->
+                where('type',"=","discussionSplit")->
+                where('content',"like","%".$event->discussion->id."-%")->
+                take(10)->
+                get(); 
+        if(count($allPosts) == 1){
```

Comment:
Instead of using these raw php methods, try to stick as close to Laravel as possible.

That means: using the (Eloquent) collection functionality; in this case (see next comment) I would use `!$allPosts->isEmpty()`